src/test/java/racingcar/domain/RaceTest.java

```diff
@@ -6,27 +6,24 @@
 import racingcar.util.BasicNumberGenerator;
 
 import java.util.Arrays;
-import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 class RaceTest {
-    @ParameterizedTest(name = "입력값({0})이 경기 규칙 최소값({1}) 이상이면 자동차의 이동거리({2})가 증가한다")
+    @ParameterizedTest(name = "입력값({0})이 경기 규칙 최소값({1}) 이상이면 자동차의 이동거리({2})가 증가한다. 경기기록 값이 반환된다.")
     @CsvSource(value = {"3:4:0", "4:4:1", "5:4:1"}, delimiter = ':')
     public void race_start_car_distance_increase(int input, int minimum, int distance) {
-        List<Car> cars = Arrays.asList(
-                new Car("carA"),
-                new Car("carB"),
-                new Car("carC"));
+
+        Cars cars = Cars.ofString(Arrays.asList("carA", "carB", "carC"));
 
         BasicNumberGenerator generator = new BasicNumberGenerator(input);
         MinimumRule rule = new MinimumRule(minimum, generator);
 
         Race race = new Race(cars, rule);
-        race.start();
+        RaceRecords raceRecords = race.start();
 
-        for (Car car : cars) {
-            assertThat(car.position()).isEqualTo(new Position(distance));
+        for (RaceRecord raceRecord : raceRecords.getRaceRecords()) {
+            assertThat(raceRecord.position()).isEqualTo(new Position(distance));
```

Comment:
아래와 같이 검증하는 방법도 있습니다.

```suggestion
        assertThat(raceRecords.getRaceRecords())
                .extracting(RaceRecord::position)
                .contains(new Position(distance));
```

for-loop와 같은 반복문이나 if절과 같은 조건문은 가능한 테스트 코드에서 지양하는것을 추천드립니다.